source/index.html.haml

```diff
@@ -106,7 +106,13 @@ title: CO-VIDEO 2021 - Global Video Tech Meetup
           %span
             6:00pm local
         %h4
-          Speakers coming soon!
+          Daniel Hoelbling-Inzko, Bitmovin
```

Comment:
Do you mind putting the company names in brackets, to be consistent with the other events please?

Thanks!